test/test_helper.rb

```diff
@@ -1,6 +1,11 @@
 require 'minitest/autorun'
 require 'cryptosystem'
 
+# Compatability for older MiniTest versions that come with older Rubies
+if !defined?(MiniTest::Test)
+  MiniTest::Test = MiniTest::Unit::TestCase
```

Comment:
Can we change the `MiniTest` references to `Minitest`?

https://github.com/seattlerb/minitest/blob/master/lib/minitest/test.rb#L3